src/documentation/0001-node-introduction/index.md

```diff
@@ -1,59 +1,29 @@
 ---
 title: Introduction to Node.js
-description: "This post is a getting started guide to Node.js, the server-side JavaScript runtime environment. Node.js is built on top of the Google Chrome V8 JavaScript engine, and it's mainly used to create web servers - but it's not limited to that"
-authors: flaviocopes
+description: "Getting started guide to Node.js, the server-side JavaScript runtime environment. Node.js is built on top of the Google Chrome V8 JavaScript engine, and it's mainly used to create web servers - but it's not limited to just that."
+author: flaviocopes
```

Comment:
The `authors` variable in the frontmatter is used to generate the `Contributors` section at the bottom of every page.

![image](https://user-images.githubusercontent.com/2608321/54472602-f3d42300-47f0-11e9-9ac0-97cb4c4b6840.png)

```suggestion
authors: flaviocopes
```

Reply:
Done.

Not for this PR, but: I guess I sorta understand the reason for naming specific authors, but unless this document doesn't get updates and falls horribly out-of-date, that list is pretty much guaranteed to get ridiculously long or else to not actually reflect authorship of the material. I wonder if something else can be done to provide credit, such as a footer link to an AUTHORS file. (That too is likely to be out of date or ridiculously long, but at least it won't be an in-page distraction.)

Reply:
Thanks for the input, @Trott! I absolutely agree with you. 

Will raise an issue to open discussion on this.

Reply:
Discussion continued on https://github.com/nodejs/nodejs.dev/issues/201.